include/libc64/qrand.h

```diff
@@ -18,6 +18,11 @@ u32 Rand_Next_Variable(u32* rndNum);
 f32 Rand_ZeroOne_Variable(u32* rndNum);
 f32 Rand_Centered_Variable(u32* rndNum);
 
-extern u32 gRandFloat;
+typedef union HexFloat {
+    u32 hex;
+    f32 flt;
+} HexFloat;
+
+extern HexFloat gRandFloat;
```

Comment:
This could use the `fu` union type from libultra

Reply:
Nice catch! thanks